sdk/src/main/java/io/opentelemetry/sdk/metrics/export/README.md

```diff
@@ -101,6 +101,10 @@ public final class PullExporter {
 
   private void OnPullRequest() {
```

Comment:
just noticed this capitalized method name, also.

Reply:
Done.